include/dxc/Support/dxcapi.extval.h

```diff
@@ -1,8 +1,160 @@
 #include "dxc/Support/dxcapi.use.h"
+#include "dxc/WinAdapter.h"
+#include "llvm/Support/raw_ostream.h"
+
 #include <cassert>
 #include <string>
 
 namespace dxc {
+
+class ExternalValidationHelper3 : public IDxcCompiler3 {
+
+public:
+  CComPtr<IDxcCompiler3> m_pCompiler3;
+  CComPtr<IDxcValidator> m_pValidator;
+
+  ExternalValidationHelper3() {
+    m_pCompiler3 = nullptr;
+    m_pValidator = nullptr;
```

Comment:
CComPtr has a default constructor that does the right thing, so this is redundant.